tests/FakeItEasy.IntegrationTests.VB/RefArgumentConstraintTests.vb

```diff
@@ -0,0 +1,38 @@
+﻿Imports System.Runtime.InteropServices
+Imports FluentAssertions
+Imports Xunit
+
+Public Class RefArgumentConstraintTests
+
+    Public Interface IFoo
+        Function Bar(ByRef x As String) As Integer
+        Function Baz(<Out> ByRef x As String) As Integer
+    End Interface
+
+    <Fact>
+    Public Sub Constraint_On_Ref_Argument_Is_Honored()

```

Comment:
This is such a minor comment, but the uppercase letters on each word look odd to me.
Of course, so does all of VB.NET.
Is this standard? I notice that the tests in the two sibling files use lowercase, except when referring to a symbol.